site/src/main/.vuepress/config.js

```diff
@@ -1288,6 +1288,12 @@ var config = {
 							['Architecture/Writing Data on HDFS','使用HDFS存储数据'],
 							['Architecture/Shared Nothing Cluster','Shared-nothing 架构']
 						]
+					},
+					{
+						title: '时间序列数据库比较',
+						children: [
+							['Comparison/TSDB-Comparison','比较']
+						]
```

Comment:
Since this document will be in master branch only after this PR merged, I don't suggest to modify the v0.11 and v0.12 parts here. Because the website can not find this doc in 0.11 and 0.12 branches and it will cause the website build failure.

Reply:
Good point. 
What about cherry pick this pr to 0.11 and 0.12? 
Because I find that 0.11 and 0.12 has the English version of TSDB-Comparison page on the website.

Reply:
> What about cherry pick this pr to 0.11 and 0.12?

Sounds great.  But for this PR, it is ok to just update the in-process doc. I think we can modify these v0.11 and v0.12 parts in `config.js` after the cherry-pick PRs merged.

Reply:
👌